sendgrid/sendgrid.py

```diff
@@ -58,3 +57,14 @@ def api_key(self, value):
     @property
     def impersonate_subuser(self):
         return self._impersonate_subuser
+
+
+class SendGridClient(SendGridAPIClient):
+    def __init__(self, **opts):
+        super().__init__(**opts)
+
+    def send_mail(self, msg):
+        try:
+            return self.client.mail.send.post(request_body=msg.get())
+        except Exception as e:
+            print("SendGrid API returned :- "+e.__str__())

```

Comment:
If we were to use the SendGridException class, i feel this would be an appropriate place to raise such an error.